### PR TITLE
Nwis json endpoint

### PR DESCRIPTION
**Pull request by phobson**

### CODE REVIEW

**Comment by phobson** [comment]
@lucashtnguyen  ready for your review

File: dockside/__init__.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
+from .nwis import Station
+from .io import *
+
+from .tests import test, teststrict, test_nowarnings
 
 __author__ = 'Lucas Nguyen'
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```
Comment (lucashtnguyen) [inline dockside/__init__.py:7]: This is a major API change. I think we should be moving to a new major version?

File: dockside/io.py [hunk 2]
```diff
@@ -1,53 +1,165 @@
...
+
+
+def _parse_ts(ts, daily):
+    """
+    Parses a single `timeSeries` object in an NWIS JSON response in a dataframe
+    """
+
+    param = ts['variable']['variableName']
+    if daily:
+        stat = ts['variable']['options']['option'][0]['value']
```
Comment (lucashtnguyen) [inline dockside/io.py:77]: Previously there was a way to get all of the different station variables from the text scraping (e.g.:
        'flow': 'cb_00060=on&',
        'temp': 'cb_00010=on&',
        'stage': 'cb_00065=on&',
        'precip': 'cb_00045=on&')
Does the JSON return everything available?

**Review by lucashtnguyen — COMMENTED** [review]
This looks great, Paul. The only issue I see is working out the versioning. What do you think?

File: dockside/io.py (hunk 2 above)
Comment (phobson) [reply dockside/io.py:77]: JSON returns everything that's available. you can use `**kwargs` to limit it

File: dockside/__init__.py (hunk 1 above)
Comment (phobson) [reply dockside/__init__.py:7]: bumped to 0.1.0

**Pull request merged**